here-naksha-lib-psql/src/commonTest/kotlin/naksha/psql/TestPsql.kt

```diff
@@ -41,7 +46,7 @@ class TestPsql {
         assertTrue(naksha_transactions.exists(), "$VIRT_TRANSACTIONS should exist!")
     }
 
-    @Test
+    //    @Test
```

Comment:
I think a TODO would help reminding for later

Reply:
Missed that one, will try to resolve / add TODO if it will need more work